.gitlab/deploy/runner/kubernetes/minikube/README.md

```diff
@@ -0,0 +1,4 @@
+# Deploy Gitlab runner on Minikube
+
+Run the `deploy` target to install the Gitlab runner using Helm. This target assumes Helm Tiller is
+already installed.
```

Comment:
Where is the runner installed to ?
Based on the directory name I suspect that it is the local minikube, the same one used to normally deploy the operator to, right ?

Reply:
Oh, forgot something, came to me looking over the main PR description ... Should possibly mention in some global README that this depends on having a local gitlab runner deployed. It feels to be that way, at least (what with bazel target and scripts added, to deploy such).

Reply:
So, right now we are using shared runners on gitlab.com. I was initially experimenting with a runner running on a local Minikube and thought that it would be useful to have some scripts to help us deploying and using it. I realized now after the review that we don't need specific instructions to Minikube. I'll keep the, almost same, script but as a generic installer to k8s. Once we decide/need to run custom runners as privileged, then we already have the scripts. We can do it on whatever long-running cluster.

Reply:
ok